Frank van der Loo (18 Jul 2004)
	MPlayer doesn't work with the latest LIVE.COM streaming because
	the destructor ~UsageEnvironment() from LIVE.COM is now protected.
	Now the public method "reclaim()" has to be used. So, instead of 
	"delete env", now "env->reclaim()" has to be used in demux_rtp.cpp.

--- ./libmpdemux/demux_rtp.cpp.orig	2004-10-13 10:09:53.000000000 -0300
+++ ./libmpdemux/demux_rtp.cpp	2004-10-13 10:10:27.000000000 -0300
@@ -407,7 +407,7 @@
   delete rtpState->sdpDescription;
   delete rtpState;
 
-  delete env; delete scheduler;
+  env->reclaim(); delete scheduler;
 }
 
 ////////// Extra routines that help implement the above interface functions:
